srcpkgs/sublime-text4/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'sublime-text4'
+pkgname=sublime-text4
+version=4107
+revision=1
+depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
+short_desc="Sophisticated text editor for code, markup and prose"
+maintainer="DirectorX <void.directorx@protonmail.com>"
+license="custom:Proprietary"
+homepage="https://www.sublimetext.com/"
+replaces="sublime-text3>=0"
```

Comment:
I don't thought about how to deal with subl3 yet.

Reply:
I think you can just mark it as:
```
conflicts="sublime-text3"
```